<?php namespace CTIMT\MyOrm\Exception\Platform;

/**
 * Generated by PHPUnit_SkeletonGenerator on 2016-05-26 at 22:48:24.
 */
class BadFunctionCallExceptionTest extends \PHPUnit_Framework_TestCase
{

    /**
     * @var BadFunctionCallException
     */
    protected $object;

    /**
     * Sets up the fixture, for example, opens a network connection.
     * This method is called before a test is executed.
     */
    protected function setUp()
    {
        $this->object = new BadFunctionCallException;
    }

    /**
     * Tears down the fixture, for example, closes a network connection.
     * This method is called after a test is executed.
     */
    protected function tearDown()
    {
        
    }
}
